dashboard/app/views/admin_users/user_projects_form.html.haml

```diff
@@ -0,0 +1,72 @@
+%h1 User Projects
+
+%p This page allows an admin to view active and deleted projects, and restore deleted projects for a user. The projects are sorted by most recently updated first. All times in UTC.
+
+:ruby
+  def format_time(datetime)
+    datetime.try(:strftime, '%F %T')
+  end
+
+= form_tag url_for(action: 'user_projects_form'), method: 'get', class: 'form-inline', enforce_utf8: false do
```

Comment:
Could there be some indication in the description (or an example?) of what this form takes in?

Reply:
On the next line there is a `text_field_tag` which describes that the field takes in text. I'll add a description to the `user_projects_form` action to make this more clear